root/artist/header.tt

```diff
@@ -1,4 +1,6 @@
-<div class="artistheader">
+[% header = (html_escape(artist.l_type_name) or l('Artist')) _ 'header' %]
```

Comment:
Two issues here:
- Localized type name (using either `l_` prefix or `l` function) for class identifier will break for non-English users
- *Other* type is not handled properly
I suggest to combine classes rather than to replace them. Stylesheet will be cleaner (Only need to set `background-image`.) and more robust (Other cases will be auto-handled.).